js/listview.js

```diff
@@ -43,6 +48,16 @@ enyo.kind({
 		}
 	},
 
+	rendered:function() {
+
+		this.inherited(arguments);
+		this.scrollbar_session_value = preferences.getOptions("scroll-value");
```

Comment:
Options in the `preferences` object is reserved for settings options of the user (specifically options coming from the Privacy dialog, see [here](https://github.com/llaske/sugarizer/blob/dev/js/dialog.js#L1014) for an example of use).
So instead of using it I suggest to create a new property and new setter/getter in the `preferences` object.